### ISSUE TITLE
Add code formatter

### ISSUE THREAD

**Issue by annavik** [annavik/websustainability#15]
To auto format code using a shared project config for tab size, etc. For example we can use [Prettier](https://prettier.io/)?

**Comment by c0ldbear** [comment]
Yeah, that could be nice! Is there any specific settings file we need or just a default one maybe? :)
